zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -23,15 +24,11 @@
 
 @document_enum
 class EcommerceCrawlStrategy(str, Enum):
-    full: str = "full"
-    """Follow most links within the domain of URL in an attempt to discover and
-    extract as many products as possible."""
-
-    navigation: str = "navigation"
+    default: str = "default"
```

Comment:
I think we should start with keeping it backwards compatible - offer all 3 strategies for now.

Reply:
We can handle deprecation later.

Reply:
Or does it require too much noise in the spider?

Reply:
Updated the PR to still include the existing `full` + `navigation` strategies but then updated their description to mention they're deprecated.